keyboards/preonic/keymaps/boy314/keymap.c

```diff
@@ -0,0 +1,223 @@
+/* Copyright 2015-2017 Jack Humbert
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "preonic.h"
+#include "action_layer.h"
+
+#define FN_CAPS LT(_FL, KC_CAPSLOCK)
+
+// define layers
+#define _QWERTY 0
+#define _DVORAK 1
+#define _OSU 2
+#define _LOWER 10
+#define _RAISE 11
+#define _ADJUST 12
+
+// macro shortcuts
+#define QWERTY TO(_QWERTY)
+#define DVORAK TO(_DVORAK)
+#define OSU TO(_OSU)
+#define LOWER M(_LOWER)
+#define RAISE M(_RAISE)
+
+#define _______ KC_TRNS
+#define XXXXXXX KC_NO
+ 
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+/* Qwerty
+ * ,-----------------------------------------------------------------------------------.
+ * | Esc  |   1  |   2  |   3  |   4  |   5  |   6  |   7  |   8  |   9  |   0  | Bksp |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | Tab  |   Q  |   W  |   E  |   R  |   T  |   Y  |   U  |   I  |   O  |   P  | Enter|
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * | Caps |   A  |   S  |   D  |   F  |   G  |   H  |   J  |   K  |   L  |   ;  |   '  |
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * | Shift|   Z  |   X  |   C  |   V  |   B  |   N  |   M  |   ,  |   .  |   /  | Shift|
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | Ctrl |  Del | Alt  | GUI  |Lower |    Space    |Raise | Left | Down |  Up  |Right |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_QWERTY] = {
```

Comment:
Could you update these so they're using this format:
https://github.com/qmk/qmk_firmware/blob/master/keyboards/preonic/keymaps/default/keymap.c#L53